src/plugins/intel_cpu/tests/functional/test_utils/cpu_test_utils.hpp

```diff
@@ -4,13 +4,15 @@
 
 #pragma once
 
-#include <string>
-#include "ie_system_conf.h"
+#include "openvino/runtime/compiled_model.hpp"
+#include "openvino/runtime/exec_model_info.hpp"
+#include "openvino/runtime/system_conf.hpp"
 #include "shared_test_classes/base/layer_test_utils.hpp"
```

Comment:
seems need to remove

Reply:
Seems we cannot remove it, because many files reference it, which will cause compilation failure.